### ISSUE TITLE
Implement rot90

### ISSUE THREAD

**Issue by simoami** [nicolaspanel/numjs#28]
ref: https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.rot90.html

Using numjs to drive a 16x16 led matrix and would like to rotate view based on device orientation.

```
> var m = numjs.array([[1,2],[3,4]], int)
> m
array([[1, 2],
       [3, 4]])

> numjs.rot90(m)
array([[2, 4],
       [1, 3]])

> numjs.rot90(m, 2)
array([[4, 3],
       [2, 1]])

> var m = numjs.arange(8).reshape(2,2,2)
> numjs.rot90(m, 1, [1,2])
array([[[1, 3],
        [0, 2]],
```

**Comment by nicolaspanel** [comment]
Thank you for your interest in numjs but I don't have enough time right now to work on such features. If you do, PR are always welcome ;)
Best regards

**Comment by nicolaspanel** [comment]
BTW it should not be very difficult to implement using ndarray#lo function (see https://github.com/scijs/ndarray#slicing)

**Comment by tryshchenko** [comment]
I have managed to implement all cases except one: method supports array and amount of iterations. (two arguments). Custom axis (third argument) is not implemented. Does it worth to submit a PR without it?

**Comment by nicolaspanel** [comment]
Of course, send the PR and I will try to fix the last case. Thanks

**Comment by tryshchenko** [comment]
@nicolaspanel I've opened a PR https://github.com/nicolaspanel/numjs/pull/30
There are feature and unit tests for it.
I'm concerned a little bit about pushing /dist folder (is it done on purpose) but decided to avoid doing something in this direction in PR for rot90.

**Comment by nicolaspanel** [comment]
implemented in #32
